paddle/inference/inference.cc

```diff
@@ -62,7 +80,7 @@ void InferenceEngine::LoadInferenceModel(
 }
 
 bool InferenceEngine::IsParameter(const framework::VarDesc* var) {
-  if (var->Persistable()) {
+  if (var->Persistable() && var->Name() != "feed" && var->Name() != "fetch") {
```

Comment:
Agree. And we don't need to check `fetch` because fetch will not be an input to an op.
We can get the feed var name from the feed op's input info.

Will fix in the future PR.